src/to-do.js

```diff
@@ -0,0 +1,308 @@
+          toDoSectionPrority.setAttribute('class', 'medium-priority');
+          toDoSectionPrority.innerHTML = toDoList[i].priority;
+          todoSection.appendChild(toDoSectionPrority);
+        } else if (toDoList[i].priority === 'Low') {
+          const toDoSectionPrority = document.createElement('p');
+          toDoSectionPrority.setAttribute('class', 'low-priority');
+          toDoSectionPrority.innerHTML = toDoList[i].priority;
+          todoSection.appendChild(toDoSectionPrority);
+        }
+
+        const statusButton = document.createElement('button');
+        statusButton.innerHTML = 'Change Status';
+        statusButton.setAttribute('class', 'btn btn-info');
+        todoSection.appendChild(statusButton);
+
+        statusButton.addEventListener('click', () => {
+          if (toDoList[i].status === 'Complete') {
+            toDoList[i].status = 'Pending';
+
+
+            saveData(toDoList);
+
+            localStorage['todo-library-data'] = JSON.stringify(toDoList);
+
+            const allContent = document.querySelector('.all-content');
+            allContent.innerHTML = '';
+            navBar();
+            allContent.appendChild(displayToDo(name, value));
+          } else if (toDoList[i].status === 'Pending') {
+            toDoList[i].status = 'Complete';
+
+            saveData(toDoList);
+
+            localStorage['todo-library-data'] = JSON.stringify(toDoList);
+
+            const allContent = document.querySelector('.all-content');
+            allContent.innerHTML = '';
+            navBar();
+            allContent.appendChild(displayToDo(name, value));
+          }
+        });
+
+        const toDoObject = toDoList[i];
+
+        const editIcon = document.createElement('i');
+        editIcon.setAttribute('class', 'fas fa-edit');
+
+        editIcon.addEventListener('click', () => {
+          editToDo(toDoList.indexOf(toDoObject));
+        });
+
+        const deleteIcon = document.createElement('i');
+        deleteIcon.setAttribute('class', 'fas fa-trash-alt');
+
+        deleteIcon.addEventListener('click', () => {
+          deleteToDo(toDoList.indexOf(toDoObject));
+        });
+
+        todoSection.appendChild(editIcon);
+        todoSection.appendChild(deleteIcon);
+
+        toDoDiv.appendChild(todoSection);
+
+        toDoSectionMainDiv.appendChild(todoSection);
+      }
+    }
+
+
+    addToDoButton.addEventListener('click', () => {
+      displayToDoForm(name, value);
+    });
+  };
+
+  const displayToDoForm = (name, value) => {
+    const toDoForm = document.createElement('div');
+    toDoForm.setAttribute('class', 'card project-module');
+    toDoForm.setAttribute('id', 'todoDoForm');
+    const title = document.createElement('label');
+    title.setAttribute('class', 'nameTxt');
+    toDoForm.appendChild(title);
+    title.innerHTML = 'Title';
+    const titleInput = document.createElement('input');
+    titleInput.setAttribute('id', 'title-id');
+    titleInput.type = 'text';
```

Comment:
You should separate your application logic (i.e. creating new todos, setting todos as complete, changing todo priority, etc.) from the DOM-related stuff, so keep all of those things in separate modules.